gulpjs/examples/build/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "build",
+  "version": "1.0.0",
+  "description": "",
+  "main": "gulpfile.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
Please use the Apache License. https://github.com/GoogleCloudPlatform/cloud-builders/blob/master/LICENSE